### DISCUSSION TITLE
Seek advice for eligible Chart control

### DISCUSSION THREAD

**Discussion by sigmarsson** [unoplatform/uno#8405] [Q&A]
Which chart control could you recommend to employ, which is eligible and works properly for at least the subset of given Uno heads, such as 

- Windows Desktop 10/11
- Web browser
- Android

Alternatives are welcome from both commercial and community editions.

**Comment by francoistanguay** [comment]
Have you checked https://www.infragistics.com/products/uno-platform ?

**Reply by sigmarsson** [reply]
That is in alpha, was harmonized with WinUI 0.5 API but since then it has been abandoned or suspneded for a long time.

**Reply by francoistanguay** [reply]
Would you have what you need in https://github.com/beto-rodriguez/LiveCharts2 if it was available for Uno?

**Reply by sigmarsson** [reply]
Yes, I seek a chart control candidate to display line chart. Infragistics had started to introduce Ultimate UI for the Uno flavor but never released. I witness Telerik has bogged down as well somewhere on the road. [This one](https://platform.uno/blog/lightningchart-introduces-uno-platform-support/) does not support WASM and basically it runs inside a WebView2 only. As it is pure JS. If I get it right. What you highlighted; IG would be the best candidate.

**Reply by sigmarsson** [reply]
yes, as long as LiveCharts2 works on those Uno heads.

**Reply by sigmarsson** [reply]
@francoistanguay please keep posted if

- There will be a Chart control for at least Windows Desktop 10/11
- There will be efforts to update Infragistics Chart control to harmonize again with WinUI 1.0 API (it worked with WinUI 0.5)
- There will be efforts to introduce LiveChart2 into Uno 

and of course, when. IF Uno has not the capacity I can purchase the control, the question which vendor.

**Reply by francoistanguay** [reply]
I will keep you posted. As LiveCharts alreadys works with UWP targetting Skia, it`s matter of spending a few cycles to make sur it works with WinUI.

**Comment by sigmarsson** [comment]
@francoistanguay could you please estimate when the chart will be available ?

**Reply by jeromelaban** [reply]
The author of LiveCharts has pushed a new preview of his Uno compatible package: https://www.nuget.org/packages/LiveChartsCore.SkiaSharpView.Uno.

**Reply by sigmarsson** [reply]
Thank you @jeromelaban @francoistanguay .
